client/account-recovery/forgot-username/forgot-username-form/test/index.jsx

```diff
@@ -57,83 +48,60 @@ describe( 'ForgotUsername', () => {
+					userData={ {
+						firstName: '',
+						lastName: 'Bar',
+						url: 'test.example.com',
+					} }
+				/> );
 
 			// Expect the button to be disabled
 			expect( wrapper.find( '.forgot-username-form__submit-button' ).prop( 'disabled' ) ).to.be.ok;
 		} );
 
-		it( 'should be disabled if last name field is blank', function() {
-			const wrapper = mount( <ForgotUsernameFormComponent className="test__test" /> );
-
-			// Initialize all inputs with some dummy text
-			inputSelectors.forEach( selector => {
-				wrapper.find( selector ).node.value = 'dummy text';
-				wrapper.find( selector ).simulate( 'change' );
-			} );
-
-			// Change the field to be empty
-			wrapper.find( '.forgot-username-form__last-name-input' ).node.value = '';
-			wrapper.find( '.forgot-username-form__last-name-input' ).simulate( 'change' );
+		it( 'should be disabled if lastName is blank', function() {
+			const wrapper = mount(
+				<ForgotUsernameFormComponent className="test__test"
+					userData={ {
+						firstName: 'Foo',
+						lastName: '',
+						url: 'test.example.com',
+					} }
+				/> );
 
 			// Expect the button to be disabled
 			expect( wrapper.find( '.forgot-username-form__submit-button' ).prop( 'disabled' ) ).to.be.ok;
 		} );
 
-		it( 'should be disabled if site url field is blank', function() {
-			const wrapper = mount( <ForgotUsernameFormComponent className="test__test" /> );
-
-			// Initialize all inputs with some dummy text
-			inputSelectors.forEach( selector => {
-				wrapper.find( selector ).node.value = 'dummy text';
-				wrapper.find( selector ).simulate( 'change' );
-			} );
-
-			// Change the field to be empty
-			wrapper.find( '.forgot-username-form__site-url-input' ).node.value = '';
-			wrapper.find( '.forgot-username-form__site-url-input' ).simulate( 'change' );
+		it( 'should be disabled if url is blank', function() {
+			const wrapper = mount(
+				<ForgotUsernameFormComponent className="test__test"
+					userData={ {
+						firstName: 'Foo',
+						lastName: 'Bar',
+						url: '',
+					} }
+				/> );
 
 			// Expect the button to be disabled
 			expect( wrapper.find( '.forgot-username-form__submit-button' ).prop( 'disabled' ) ).to.be.ok;
 		} );
 
 		it( 'should be enabled when all fields are filled in', function() {
-			const wrapper = mount( <ForgotUsernameFormComponent className="test__test" /> );
-
-			// Initialize all inputs with some dummy text
-			inputSelectors.forEach( selector => {
-				wrapper.find( selector ).node.value = 'dummy text';
-				wrapper.find( selector ).simulate( 'change' );
-			} );
+			const wrapper = mount(
+				<ForgotUsernameFormComponent className="test__test"
+					userData={ {
+						firstName: 'Foo',
+						lastName: 'Bar',
+						url: 'test.example.com',
+					} }
+				/> );
 
 			// Expect the button to be enabled
 			expect( wrapper.find( '.forgot-username-form__submit-button' ).prop( 'disabled' ) ).to.not.be.ok;
 		} );
-
-		it( 'should be disabled when clicked', function() {
```

Comment:
Should we continue testing this case? when all fields are set and `isRequesting === true`